### PR TITLE
Ch 769: Rename/Delete page variations

### PR DESCRIPTION
**Pull request by eshtadc**

### CODE REVIEW

File: acquia_lift.admin.inc [hunk 1]
```diff
@@ -930,3 +921,212 @@ function acquia_lift_page_variation_details_ajax_callback($form, &$form_state) {
...
+  );
+
+  $form_state['build_info']['args']['agent_name'] = $agent_name;
+  $form_state['build_info']['args']['variation_index'] = $variation_index;
+
+  $output = ctools_modal_form_wrapper('acquia_lift_page_variation_rename', $form_state);
+
+  if ($form_state['executed']) {
+    $settings = array();
+    $option_sets = $option_sets = personalize_option_set_load_multiple(array(), array('decision_name' => $form_state['values']['variation_set_name']));
```
Comment (katbailey) [inline acquia_lift.admin.inc:958]: Again, `$option_sets = $option_sets =` :)

File: acquia_lift.admin.inc [hunk 2]
```diff
@@ -930,3 +921,212 @@ function acquia_lift_page_variation_details_ajax_callback($form, &$form_state) {
...
+  );
+
+  $form_state['build_info']['args']['agent_name'] = $agent_name;
+  $form_state['build_info']['args']['variation_index'] = $variation_index;
+
+  $output = ctools_modal_form_wrapper('acquia_lift_page_variation_delete', $form_state);
+
+  if ($form_state['executed']) {
+    $settings = array();
+    $option_sets = $option_sets = personalize_option_set_load_multiple(array(), array('decision_name' => $form_state['values']['variation_set_name']));
```
Comment (katbailey) [inline acquia_lift.admin.inc:1062]: At least it's giving me something to comment on ;)

File: acquia_lift.admin.inc (hunk 1 above)
Comment (eshtadc) [inline acquia_lift.admin.inc:958]: Now we know where I copied it from.

**Comment by eshtadc** [comment]
Closing to make a new PR against 7.x-1.x.

**Pull request closed without merging**